website/resources/components/messaging.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Arcus Messaging
-description: Messaging with Microsoft Azure in a breeze.
+description: Arcus Messaging is a library that helps with the integration of messaging systems like queues or event subscriptions, with application message handling that describes the business.
```

Comment:
I dont understand what you mean with 'with application message handling that describes the business'.
Maybe change it to

```suggestion
description: Arcus Messaging is a library that helps with the integration of messaging systems like queues or event subscriptions and allows you to focus on implementing business logic when processing messages.
```

Reply:
This is actually a copy from our official Messaging introduction: 
![image](https://user-images.githubusercontent.com/9039753/210936332-113e7f39-eaa7-4d2f-a289-36bd1b3affa7.png)
So I better change it there, too, right?
Done in https://github.com/arcus-azure/arcus.messaging/pull/379